.github/ISSUE_TEMPLATE.md

```diff
@@ -39,6 +39,7 @@ What needs to be done to address this issue? Ideally, provide a pull request wit
 ### Additional Information
 
 * React Native version: [FILL THIS OUT: Be specific, filling out "latest" here is not enough.]
+* Node/npm version: [FILL THIS OUT: `node -v` and `npm -v`]
```

Comment:
Can do - also this may depend on https://github.com/facebook/react-native/pull/14428

We may want to wait until this one is finished.